zkevm-circuits/src/witness/tx.rs

```diff
@@ -63,12 +61,9 @@ impl Transaction {
                     Value::known(F::from(self.id as u64)),
                     Value::known(F::from(TxContextFieldTag::GasPrice as u64)),
                     Value::known(F::zero()),
-                    challenges.evm_word().map(|evm_word| {
-                        RandomLinearCombination::random_linear_combine(
-                            self.gas_price.to_le_bytes(),
-                            evm_word,
-                        )
-                    }),
+                    challenges
+                        .evm_word()
+                        .map(|evm_word| rlc::value(&self.gas_price.to_le_bytes(), evm_word)),
```

Comment:
It is a dummy variable but naming it `challenge`, `randomness`, or `r` could be less confusing.
```suggestion
                        .map(|challenge| rlc::value(&self.gas_price.to_le_bytes(), challenge)),
```